vnext/src/Libraries/Alert/Alert.windows.js

```diff
@@ -38,18 +39,33 @@ class Alert {
     // The text 'OK' should be probably localized. iOS Alert does that in native.
     const validButtons: Buttons = buttons
       ? buttons.slice(0, 3)
-      : [{text: 'OK'}];
+      : [{text: 'OK', default: true}];
     const buttonPositive = validButtons.pop();
     const buttonNegative = validButtons.pop();
     const buttonNeutral = validButtons.pop();
 
+    // Find the first button where 'default' is set to true
+    // in order of declared buttons.
+    const defaultIndex = [
+      buttonNeutral,
+      buttonNegative,
+      buttonPositive,
+    ].findIndex(b => b != null && b.default);
+
+    // XAML has an enum to specify the default button, which is:
+    //   None = 0, Primary = 1, Secondary = 2, Close = 3
+    // If no default button is found, specify 0 for None, otherwise
+    // convert the index to its corresponding XAML enum value.
+    const defaultButton = defaultIndex >= 0 ? 3 - defaultIndex : 0;
```

Comment:
Technically, we don't need to do any checks here, since native code has validation, but it's probably good measure.